src/pages/singleProduct/SingleProduct.css

```diff
@@ -0,0 +1,67 @@
+.singleProduct {
+  width: 100%;
+  max-width: 56.25rem;
```

Comment:
You can remove the width and just go with max-width to limit its width as per your value

Reply:
updated